libs/community/langchain_community/vectorstores/sqlserver.py

```diff
@@ -327,7 +379,7 @@ def _search_store(
                             ),
                         ),
                     )
-                    .filter()
+                    .filter(*filter_by)
```

Comment:
what happens if this stays empty []?

Reply:
do we have a test for this, would an accurate SQL clause get generated?

Reply:
So, the regular similarity search tests do not pass any filters, then the value is empty for filters. Those tests succeed that is how this case is tested.